src/index.js

```diff
@@ -58,4 +59,25 @@ document.getElementById('table').onkeyup = (e) => {
     list[index].Description = e.target.innerHTML;
     ToLocalStorage(list);
    }
+};
+document.getElementById('table').onchange = (e) => {
+  if (e.target.className === 'todo') {
+    if (e.target.checked) {
+      e.target.parentElement.nextElementSibling.style.textDecoration = 'line-through';
+      const index = e.target.getAttribute('data-index');
+      list[index].completed = true;
+      ToLocalStorage(list);
+    } else {
+      e.target.parentElement.nextElementSibling.style.textDecoration = 'none';
```

Comment:
- [ ] sometimes when i add and delete tasks a duplicated index will be produced , kindly fix that 🙏 

![Repetableindex](https://user-images.githubusercontent.com/89282221/215460246-f9e90d2a-89bd-4489-aa0a-911bb64e9ed1.gif)